src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.State.cs

```diff
@@ -44,7 +45,12 @@ private class State
                 /// </summary>
                 public ValueSource<Optional<Compilation>>? Compilation { get; }
 
-                public TrackedGeneratorDriver GeneratorDriver { get; }
+                /// <summary>
+                /// The best generated documents we have for the current state. If this is a final state, then this should reflect
+                /// what is actually correct for the compilation. For an in-progress state, this may represent from a previous
+                /// run (and we have since updated the inputs).
```

Comment:
not sure how to read this sentence.  something about hte "and we have since updated the inputs" is clunky/unclear.

Reply:
Rephrased to:

```
The best generated documents we have for the current state. If this is a final state, then this should reflect
what is actually correct for the compilation. For an in-progress state, this may represent the generated documents
prior to the compilation being forked; in that case the generated documents could still be used if we are looking
to use an in-progress compilation that isn't entirely correct, but we will need to rerun the generators eventually.
```